<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PTALPALL-LAP" ignored="0" name="CreateAProgramAndPublishTest" tests="1" failures="1" timestamp="22 Oct 2018 11:20:14 GMT" time="0.673" errors="0">
  <testcase name="@AfterMethod quit" time="0.599" classname="com.oracle.rsys.TestCases.CreateAProgramAndPublishTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.oracle.rsys.TestCases.CreateAProgramAndPublishTest.quit(CreateAProgramAndPublishTest.java:112)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod quit -->
  <testcase name="fnSampleProgramPublishTest" time="0.021" classname="com.oracle.rsys.TestCases.CreateAProgramAndPublishTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.relevantcodes.extentreports.model.Test.trackLastRunStatus(Test.java:390)
at com.relevantcodes.extentreports.ExtentTest.log(ExtentTest.java:102)
at com.relevantcodes.extentreports.ExtentTest.log(ExtentTest.java:188)
at com.oracle.rsys.BaseTest.BaseTestClass.fnInit(BaseTestClass.java:78)
at com.oracle.rsys.TestCases.CreateAProgramAndPublishTest.fnSampleProgramPublishTest(CreateAProgramAndPublishTest.java:60)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- fnSampleProgramPublishTest -->
</testsuite> <!-- CreateAProgramAndPublishTest -->
